### PR TITLE
Use react-hook-form vs formik

### PR DESCRIPTION
**Pull request by nathanstitt**
Formik seems to be no longer maintained and had issues with multiple checkbox fields that used the same `name` property.

This updates the fields to use react-hook-form internally via the "sundry" library https://github.com/nathanstitt/sundry

### CODE REVIEW

File: backend/app/controllers/api/v1/participant/studies_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Api::V1::Participant::StudiesController < Api::V1::BaseController
   before_action :set_study, only: [:launch, :land]
 
   def index
-    launched_studies = current_user.launched_studies.includes(study: [:researchers])
+    launched_studies = current_user.launched_studies.includes(:stages, study: [:researchers])
```
Comment (chrisbendel) [inline backend/app/controllers/api/v1/participant/studies_controller.rb:8]: Did I miss this in my study points/time work? Just for my own knowledge :)

**Review by chrisbendel — APPROVED** [review]

**Review by chrisbendel — APPROVED** [review]

**Comment by chrisbendel** [comment]
@nathanstitt one last thing that came to mind, adding

```kinetic-adm-01@mailinator.com``` to the readme file

**Comment by nathanstitt** [comment]
hehe, check last commit 😁

**Comment by nathanstitt** [comment]
oh wait, sorry you're correct! I made that change and missed checking in the file 😭 fixed!

**Comment by chrisbendel** [comment]
Looks good!! 👍 🚀

**Comment by chrisbendel** [comment]
Looks good!! 👍 🚀

**Pull request merged**